docs/community.aws.iam_managed_policy_module.rst

```diff
@@ -368,14 +368,14 @@ Examples
         state: present
 
     # Update a policy with a new default version
-    - name: Create IAM Managed Policy
+    - name: Update an IAM Managed Policy with new default version
       community.aws.iam_managed_policy:
         policy_name: "ManagedPolicy"
         policy: "{{ lookup('file', 'managed_policy_update.json') }}"
         state: present
 
     # Update a policy with a new non default version
-    - name: Create IAM Managed Policy
+    - name: Update an IAM Managed Policy with a non default version
```

Comment:
@priyadarshu Thanks for the contribution. Documentation file like this RST is autogenerated. You may want to edit the actual code to reflect these changes [here](https://github.com/ansible-collections/community.aws/blob/3e30e37d7188e75a3ef19173e6a476742ce21e8e/plugins/modules/iam_managed_policy.py#L60). 

Let me know if you need any additional help.

Reply:
@Akasurde Thanks for letting me know, I will contribute on the actual code.

Reply:
@Akasurde I have raised a new PR, should I close this PR?